go.mod

```diff
@@ -46,4 +46,4 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-go 1.18
+go 1.19
```

Comment:
no changes from running go mod tidy?

Reply:
![image](https://user-images.githubusercontent.com/34005253/198161554-0bd467b1-99bd-4c68-9b10-116fee9c02c7.png)
yes there is no changes after running go mod tidy.